Clarifying license properties

matentzn opened the pull request:
See #1563.

cthoyt commented on principles/fp-001-open.md:
```diff
@@ -18,7 +18,7 @@ OBO Foundry ontologies are resources for the entire biological and biomedical co
 ### For ontology providers
 
-OBO Foundry Ontologies MUST EITHER be released under a [Creative Commons](https://creativecommons.org/) [CC-BY license version 3.0](https://creativecommons.org/licenses/by/3.0/) or later, OR released into the public domain under [CC0](https://creativecommons.org/publicdomain/zero/1.0/). It should be clearly stated in the ontology file.
+OBO Foundry Ontologies MUST EITHER be released under a [Creative Common Attribution 3.0 International (CC BY 3.0) license or later](https://creativecommons.org/licenses/by/3.0/) or later, OR released into the public domain under [a Creative Commons CC0 public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/). The license MUST be [clearly stated](https://github.com/INCATools/o ...
```
Still possibility for confusion - what comes after http://purl.org/dc/terms/license? Maybe someone will put free text instead of the proper software URL. Could you give an example, plus a list to the controlled vocabulary that could be used?

matentzn replied:
Thank you @cthoyt, done.. Better now?

nataled reviewed (changes requested):
The new text incorrectly indicates the name of the license, which is Creative Commons Attribution 3.0 Unported. The International version did not start until 4.0. The following is the text after correcting that and other issues as well:

OBO Foundry Ontologies MUST EITHER be released under a [Creative Commons Attribution 3.0 Unported (CC BY 3.0)](https://creativecommons.org/licenses/by/3.0/) license or later, OR released into the public domain under [Creative Commons CC0 1.0 Public Domain Dedication (CC0 1.0)](https://creativecommons.org/publicdomain/zero/1.0/). The license MUST be clearly stated using the http://purl.org/dc/terms/license property followed by the URL representing the license (e.g. https://creativecommons.org/licenses/by/3.0/) in the ontology file ([OWL example](https://github.com/INCATools/ontology-development-kit/blob/master/docs/License.md)).

Please revise accordingly.

cthoyt commented:
if that's the case, then the text should also include the full title of CC BY 4.0 so users know there's a difference in addition to the other 2 examples

nataled commented:
CC BY 4.0 isn't mentioned, nor does it need to be since it isn't a requirement. That being said, I think CC BY 4.0 is a better choice of license. The Ops Committee would need to approve this change (it was decided some years ago to keep it at 3.0, but that was when 4.0 was pretty new). For such a discussion, I think it would be useful to count how many still use 3.0. If none, then the change is easy. If there are any that still use it, we could request that they update, and if all do so, the change again is easy.

matentzn commented:
Is 4.0 more permissive such that a migration from 3.0 to 4.0 is actually possible?

matentzn commented:
@nataled thanks a lot for your review; I have changed the text as per your suggestion, and also added explicitly the CC-BY 4 license an example in brackets (which is safe I would think). We can discuss recommending it officially, but currently, more than half (77) of the active ontologies use CC-BY 3.0.

nataled commented:
Yes, but that's not the point. Remember, no changes to the *meaning of* principles occur without Ops discussion. The decision from previous discussion was to use CC BY 3.0 even though CC BY 4.0 was available at the time, so that's what we say in the principle. If we don't have any ontologies using 3.0 than I don't imagine there'd be any issue making the change, but that needs verification.

matentzn commented:
Can you double check what comment you commented on, I think we got our comments crossed :D Sorry.

nataled commented:
My comment was on the permissiveness question, which I hadn't yet uploaded before you commented again.

matentzn commented:
Thank you!

kltm reviewed (commented):
The changes appear to be correct and an improvement--thank you!

I might note that for consistency (and technical correctness) that line 23's "CC-BY" should probably be "CC BY".

kltm commented:
NM--merged earlier!
